### PR TITLE
Update connection indexes

### PR DESCRIPTION
**Pull request by B-Rass**
- Improve data shown on `Connection` index
- Improve data shown on `Cable` index

### CODE REVIEW

File: app/views/cables/index.html.erb [hunk 1]
```diff
@@ -8,21 +8,23 @@
...
 
         <% table.with_column(Cable.human_attribute_name(:color), sort_by: :color) do |cable| %>
-          <%= cable.color %>
+          <%- if cable.color? %>
+            <span class="badge <%= cable.color %>">
+              <%= cable.color %>
+            </span>
+          <% else %>
+            <%= cable.color %>
+          <% end %>
```
Comment (nicolas-brousse) [inline app/views/cables/index.html.erb:27]: Does the else really necessary? If `cable.color?` is `false` then no need to print it. No?

File: app/views/cables/index.html.erb [hunk 2]
```diff
@@ -33,11 +35,18 @@
...
+        <% table.with_column(Connection.model_name.human, sort_by: :connections_count) do |cable| %>
           <%= cable.connections_count %>
         <% end %>
 
         <% table.with_column(style: "min-width: 160px; width: 160px") do |cable| %>
+          <div class="btn-group btn-group-sm" role="group" aria-label="...">
+            <%= link_to connections_edit_path(from_port_id: cable.ports.first), class: "btn btn-primary" do %>
+              <span class="glyphicon glyphicon-pencil" aria-hidden="true" data-toggle="tooltip" data-placement="left" title="<%= t("action.edit") %>"></span>
+              <span class="sr-only"><%= t("action.edit") %></span>
+            <% end %>
```
Comment (nicolas-brousse) [inline app/views/cables/index.html.erb:47]: Does the edit should open a modal?

File: app/views/connections/index.html.erb [hunk 3]
```diff
@@ -8,25 +8,56 @@
...
       </div>
-
       <%= render List::DataTableComponent.new(@connections) do |table| %>
         <% table.with_column(Connection.human_attribute_name(:port), sort_by: :"ports.id") do |connection| %>
           <%= connection.port&.id %>
         <% end %>
 
         <% table.with_column(Connection.human_attribute_name(:cable), sort_by: :"cables.name") do |connection| %>
-          <%= connection.cable&.name %>
+          <%- if connection.cable.color? %>
```
Comment (nicolas-brousse) [inline app/views/connections/index.html.erb:21]: Cable may be `nil`?  What happen if it is `nil`?

May be we should add a `&` between `cable` and `color?`?

File: app/views/connections/index.html.erb [hunk 4]
```diff
@@ -8,25 +8,56 @@
...
         <% end %>
 
         <% table.with_column(Connection.human_attribute_name(:paired_connection)) do |connection| %>
           <%= connection.paired_connection&.id %>
+          <%- if connection.paired_connection %>
+            <span class="pull-right">
+              <%= link_to connections_edit_path(from_port_id: connection.port), class: "btn btn-primary btn-xs" do %>
+                <span class="glyphicon glyphicon-pencil" aria-hidden="true" data-toggle="tooltip" data-placement="left" title="<%= t("action.edit") %>"></span>
+                <span class="sr-only"><%= t("action.edit") %></span>
+              <% end %>
```
Comment (nicolas-brousse) [inline app/views/connections/index.html.erb:37]: Does the edit should open a modal?

File: app/views/connections/index.html.erb [hunk 5]
```diff
@@ -8,25 +8,56 @@
...
+        <% table.with_column(PortType.model_name.human) do |connection| %>
+          <%= connection.port_type&.name %>
+        <% end %>
+
+        <% table.with_column(style: "min-width: 84px; width: 84px") do |connection| %>
+          <div class="btn-group btn-group-sm" role="group" aria-label="...">
+            <%= link_to connections_edit_path(from_port_id: connection.port), class: "btn btn-primary" do %>
+              <span class="glyphicon glyphicon-pencil" aria-hidden="true" data-toggle="tooltip" data-placement="left" title="<%= t("action.edit") %>"></span>
+              <span class="sr-only"><%= t("action.edit") %></span>
+            <% end %>
```
Comment (nicolas-brousse) [inline app/views/connections/index.html.erb:59]: Does the edit should open a modal?

**Review by nicolas-brousse — COMMENTED** [review]
@B-Rass here is my review

File: app/views/cables/index.html.erb (hunk 1 above)
Comment (B-Rass) [reply app/views/cables/index.html.erb:27]: I did it for a reason. I have to check why

File: app/views/cables/index.html.erb (hunk 2 above)
Comment (B-Rass) [reply app/views/cables/index.html.erb:47]: I don't think so

File: app/views/connections/index.html.erb (hunk 4 above)
Comment (B-Rass) [reply app/views/connections/index.html.erb:37]: I don't think so

File: app/views/connections/index.html.erb (hunk 3 above)
Comment (B-Rass) [reply app/views/connections/index.html.erb:21]: Normally there can't be a connection if there is no cable

File: app/views/cables/index.html.erb (hunk 1 above)
Comment (B-Rass) [reply app/views/cables/index.html.erb:27]: I couldn't find the reason why I did it like this. I test and check and yes I'll do as you say

**Pull request merged**